src/components/SarcophagusCreate/ArchaeologistSelect/ArchaeologistTableRow.tsx

```diff
@@ -39,11 +41,14 @@ const ArchaeolgistTableRow = ({
   const isDiggingFeeLess = archaeologist.minimumDiggingFee.lte(
     ethers.utils.parseEther(values.diggingFee.toString())
   );
-  const isDisabled = !isBountyLess || !isDiggingFeeLess;
+  const isArchaeologistMaxResValid = isMaxResurrectionTimeValid(archaeologist.maximumResurrectionTime.toNumber(), Number(values.resurrectionTime))
+
+  const isDisabled = !isBountyLess || !isDiggingFeeLess || !isArchaeologistMaxResValid;
   // checks freebond is greater
   const isFreeBondGreater = archaeologist.freeBond.gte(ethers.utils.parseEther(archTotal));
-
+  
   const selectArchaeologist = () => {
+    if(isDisabled) return
```

Comment:
@Da-Colon -- one thing to test (this maybe fixed now) --- whenever I was selecting an archaeologist from the list, I had to click it twice to select it.